packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/mcp/formatters.py

```
@@ -220,6 +287,204 @@ def format_conflict_analysis(conflicts: dict[str, Any]) -> str:
+                "total_conflicts": len(conflict_index),
+                "max_displayed": len(conflict_index),
+                "can_expand_documents": True,
+                "expand_tool": "expand_document"
+            }
+        }
+
+    @staticmethod
+    def _extract_conflicting_statements(conflict_info: dict[str, Any]) -> list[dict[str, str]]:
+        """Extract actual conflicting statements from structured conflict data."""
+        statements = []
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)